content_scripts/ui.js

```diff
@@ -37,10 +37,10 @@ ui.checkInjectedElements = () => {
       return
     const importExportEl = document.createElement('div')
     importExportEl.id = 'iondvImportExport'
-    importExportEl.setAttribute('style', 'padding-left: 10px;padding-right: 10px')
+    importExportEl.setAttribute('style', 'padding-left: 10px;padding-right: 10px; padding-top: 8px;')
```

Comment:
manually added to current version. Thanks.

Reply:
perfect then I close this PR